src/types/begin_block.rs

```diff
@@ -0,0 +1,51 @@
+use crate::proto::abci::{Event as ProtoEvent, RequestBeginBlock, ResponseBeginBlock};
+use crate::types::{Event, Evidence, Header, LastCommitInfo};
+
+#[derive(Debug, Default)]
```

Comment:
Not sure what the point of these wrapper types is -- why not just use the protobuf types?